lua/lvim/utils/git.lua

```diff
@@ -78,12 +80,50 @@ function M.update_base_lvim()
     return
   end
 
+  -- Get the messages before merging
+  local _, stdout =
+    git_cmd { args = { "--no-pager", "log", "--pretty=format:* %h -%d %s (%cr) <%an>", "HEAD..@{upstream}" } }
+
+  local function set_syntax_hl()
+    vim.cmd [[highlight LvimInfoIdentifier gui=bold]]
+    vim.cmd [[highlight link LvimInfoHeader Type]]
+    vim.fn.matchadd("LvimInfoHeader", "changelog:")
+    vim.fn.matchadd("LvimInfoHeader", "[0-9a-f]\\{7,8}")
+  end
+
+  local content_provider = function(popup)
+    local content = {}
+
+    for _, section in ipairs {
+      banner,
+      { "" },
+      { "" },
+      { "changelog: " },
+      { "" },
+      stdout,
+      { "" },
+    } do
+      vim.list_extend(content, section)
+    end
+
+    return text.align_left(popup, content, 0.1)
+  end
+
+  local Popup = require("lvim.interface.popup"):new {
+    win_opts = { number = false },
+    buf_opts = { modifiable = false, filetype = "lspinfo" },
+  }
+
   ret = git_cmd { args = { "merge", "--ff-only", "--progress" } }
   if ret ~= 0 then
     Log:error("Update failed! Please pull the changes manually in " .. get_lvim_base_dir())
     return
   end
 
+  -- Display the window after merging
+  Popup:display(content_provider)
```

Comment:
There's already a place reserved where you can call this function 😅 

https://github.com/LunarVim/LunarVim/blob/c015d0feeed35b34d84558a8e4a49549b3b67781/lua/lvim/utils/hooks.lua#L78-L79

Reply:
what's why I opened a draft PR, I'm not sure about the placement of the my code. and I funny enough I didn't see the TODO